help/en/html/apps/DecoderPro/resources/decoder-template.xml

```diff
@@ -13,7 +13,7 @@
 <!-- FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License  -->
 <!-- for more details.                                                      -->
 
-<decoder-config xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://jmri.org/xml/schema/decoder.xsd" showEmptyPanes="no" >
+<decoder-config xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://jmri.org/xml/schema/decoder.xsd" showEmptyPanes="no" >
```

Comment:
Links in xmlns entries should not be updated; they are as much _names_, as URL s, and the names should remain constant (there is, in fact, no requirement that a URL in an xmlns be resolvable, just that it have a unique meaning).